src/net.cpp

```diff
@@ -1904,7 +1911,10 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
         // these conditions are met, check the nNextFeeler timer to decide if
         // we should open a FEELER.
 
-        if (nOutboundFullRelay < m_max_outbound_full_relay) {
+        if (!m_anchors.empty() && (nOutboundBlockRelay < m_max_outbound_block_relay)) {
```

Comment:
I think comment above isn't accurate anymore as it clearly mentions OUTBOUND_FULL_RELAY has the highest priority which isn't true after this patchset.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-700159056).